atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -345,14 +345,14 @@ def get_registry_session(self, registry):
         return RegistrySession(registry, insecure=insecure, dockercfg_path=secret_path)
 
     def run(self):
-        digests = set()
+        digests = list()
```

Comment:
Why change it to a `list`? I believe `set` was used to ensure no duplicates are added.

Reply:
`dict` (which is Manifest Digest) is an unhashable type. I'm not sure if duplicates are possible in this list - and if there is a nice way to remove those

Reply:
Can we make `ManifestDigest` a hashable type?

Reply:
Its a bit complicated, but possible, I think. I believe it would use tuple as a base and have init part reworked - I'll try to implement this in a new commit, stay tuned

Reply:
That was surprisingly easier that I thought.

Introduced a new `HashableDict` class, which calculates a hash by converting dict items into `frozenset`. This puts several limitations - `ManifestDigest` won't change the hash if any values were updated after it was put in the `set`, but luckily we don't need to do this.